app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -417,6 +426,18 @@ class ProjectInformationActivity : AppCompatActivity() {
         }, { showToast(getString(R.string.could_not_download_video), Toast.LENGTH_LONG) })
     }
 
+    private fun deletingAndCopyingVideo(fileUrl: String, deleteDirectory: File, copyDirectory: File){
+        val file = File(deleteDirectory, fileDB.getFileName(fileUrl))
+        if(file.exists()){
+            val newFile = File(copyDirectory, fileDB.getFileName(fileUrl))
+            file.copyRecursively(newFile, true)
+            file.deleteRecursively()
```

Comment:
Instead of copying then deleting the file, just moving the file could be cleaner.
For example using Files.move(fromFile, toFile) from com.google.common.io.Files.

Reply:
Yes my bad I forgot to push it this way yesterday